google/cloud/compute/CMakeLists.txt

```diff
@@ -116,114 +175,154 @@ set(service_dirs
     "v1/"
```

Comment:
We should just remove this from the list. There is nothing in here.

https://github.com/googleapis/google-cloud-cpp/tree/main/google/cloud/compute/v1

Then we can remove all the special cases checking `if (NOT "${dir}" STREQUAL "v1/")` below.